huxley/www/js/markdown/AdvisorProfileView.md

```diff
@@ -0,0 +1,9 @@
+## Profile
+
+Welcome, {{ first_name }}! We are very excited to see {{ school_name }} at BMUN {{ conference_session }} this year! Here, you can view and edit your registration information for the conference. Please note that fees are currently **estimates** based on the approximate delegation size given during registration. You can find more information on our fees [here](http://www.bmun.org/conference-fees/).
```

Comment:
May as well make these lowerCamelCase so that the props can be consistently named with the rest of the app, e.g. `firstName`, `schoolName`, etc.